src/components/LoanFinding/LendingCategorySection.vue

```diff
@@ -78,7 +78,6 @@ export default {
 		title: {
 			type: String,
 			default: '',
-			required: true
```

Comment:
The prop can also be removed from the component where it's being used.

Reply:
Fixed, thanks!